backend/src/apiserver/server/test/pipeline_plus_component/component.yaml

```diff
@@ -0,0 +1,9 @@
+name: whalesay
```

Comment:
These all look like test data, so I suggest we put them under /apiserver/test/testdata/ directory. Bazel can then include testdata using glob rule.

Reply:
I'm mostly following the existing convention (to minimize the PR diff size and make reviewing easier). See https://github.com/kubeflow/pipelines/tree/master/backend/src/apiserver/server/test

I'm totally fine with moving all that test data to a different directory, but I think it should be done in a separate refactoring PR.

Reply:
BTW, this file itself is not used (only the archive is), but the other test cases had unpacked files, so I'm following their example.

Reply:
Since you're making the change now anyway, can you do the cleanup and move test data to a testdata/ directory please?

Reply:
Is it OK if I do this is a separate PR so that it's more clean? I'd really like to separate refactorings from changes.

Reply:
Sure, SGTM.